data/json/recipes/other/power_supplies.json

```diff
@@ -290,6 +290,40 @@
       [ [ "pipe", 20 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "makeshift_wind_turbine",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "fabrication",
+    "skills_required": [ "electronics", 3 ],
+    "difficulty": 3,
+    "time": "1 h",
+    "reversible": true,
+    "decomp_learn": 2,
+    "autolearn": true,
+    "book_learn": [ [ "manual_electronics", 7 ], [ "textbook_electronics", 6 ], [ "advanced_electronics", 6 ] ],
```

Comment:
If it's autolearned at fab/elec 3, then a book learn of 6 or 7 makes no sense

Reply:
I completely copied it from the turbine. Maybe that should be changed too. 
I'll lower the book learn.